src/api/src/helper.rs

```diff
@@ -577,9 +573,8 @@ pub fn pb_value_to_value_ref<'a>(
         ValueData::BinaryValue(bytes) => ValueRef::Binary(bytes.as_slice()),
         ValueData::StringValue(string) => ValueRef::String(string.as_str()),
         ValueData::DateValue(d) => ValueRef::Date(Date::from(*d)),
-        ValueData::DatetimeValue(d) => ValueRef::DateTime(DateTime::new(*d)),
         ValueData::TimestampSecondValue(t) => ValueRef::Timestamp(Timestamp::new_second(*t)),
-        ValueData::TimestampMillisecondValue(t) => {
+        ValueData::DatetimeValue(t) | ValueData::TimestampMillisecondValue(t) => {
```

Comment:
Oh, we should map to microsecond according to https://github.com/GreptimeTeam/greptimedb/issues/5489